app/controllers/employees_controller.rb

```diff
@@ -6,6 +6,12 @@ def new
   def create
     @employee = Employee.new(employee_params)
 
+    unless EmployeeFinder.new.employee_exists?(@employee.slack_username)
+      flash.now[:error] = "\"#{@employee.slack_username}\" is not a valid employee (that username was not found)."
```

Comment:
how about just `"There is not a slack user with the username \"#{@employee.slack_username}\" in your organization"`